l10n_es_intrastat_report/migrations/14.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    cr.execute(
```

Comment:
Use `rename_fields` openupgradelib method for also handling possible appearances in export profiles, etc.